### PR TITLE
Solve day08 in python

### PR DESCRIPTION
**Pull request by Stektpotet**

### CODE REVIEW

File: 2021/day08/io/example.output [hunk 1]
```diff
@@ -1,3 +1,2 @@
 26
 61229
-
```
Comment (Stektpotet) [inline 2021/day08/io/example.output]: White space being the biggest annoyance as of yet this AoC🙃

Comment (Arxcis) [reply 2021/day08/io/example.output]: Hmm :thinking:

Comment (Arxcis) [reply 2021/day08/io/example.output]: Sorry . It should not be a problem in most cases, but I have done some mistakes in some of the `example.input` and `.output` files.

**Review by Arxcis — APPROVED** [review]
Najs solution :+1: :clap: :smile:

**Pull request merged**